src/webviews/extension-side/serviceRegistry.node.ts

```diff
@@ -105,4 +116,9 @@ export function registerTypes(serviceManager: IServiceManager, _isDevMode: boole
     );
 
     serviceManager.addSingletonInstance<IExtensionSideRenderer>(IExtensionSideRenderer, new ExtensionSideRenderer());
+
+    serviceManager.addSingleton<ISurveyBanner>(ISurveyBanner, DataScienceSurveyBanner);
+    serviceManager.addBinding(ISurveyBanner, IExtensionSingleActivationService);
+    // Activation Manager
+    serviceManager.add<IExtensionActivationManager>(IExtensionActivationManager, ExtensionActivationManager);
```

Comment:
WorkspaceActivation, Banners, and ActivationManager registered through `extension-side`.